torchrec_dlrm/requirements.txt

```diff
@@ -0,0 +1,3 @@
+fbgemm-gpu
+importlib-metadata<5.0.0
+torchrec-nightly
```

Comment:
this is fine for now, but let's add a pinned version soon (we have our next release for fbgemm-gpu and TorchRec coming out Friday this week)

Reply:
Sure, I also think we should pin all these (also applies to the base container tag). And probably use [torchrec](https://pypi.org/project/torchrec/) instead of[ torchrec-nightly](https://pypi.org/project/torchrec-nightly/) in the end

Reply:
I just left versions open for now as this is still in development